include/envoy/http/header_map.h

```diff
@@ -627,21 +630,33 @@ using HeaderMapPtr = std::unique_ptr<HeaderMap>;
  */
 
 // Base class for both request and response headers.
-class RequestOrResponseHeaderMap : public virtual HeaderMap {};
+class RequestOrResponseHeaderMap : public virtual HeaderMap {
+public:
+  INLINE_REQ_RESP_HEADERS(DEFINE_INLINE_HEADER)
+};
 
 // Request headers.
-class RequestHeaderMap : public RequestOrResponseHeaderMap {};
+class RequestHeaderMap : public RequestOrResponseHeaderMap {
```

Comment:
The asymmetry here is that there are no inline request headers/trailers, right? (there is no `RequestHeaderOrTrailerMap` to extend from)

Reply:
Right